src/types/Synergism.d.ts

```diff
@@ -347,6 +348,7 @@ export interface Player {
     autoFortifyToggle: boolean,
     autoEnhanceToggle: boolean,
     autoResearchToggle: boolean,
+    autoResearchMode: AutoResearchMode,
```

Comment:
in CheckVariables.ts please add a default value, ie:
```ts
if (data.autoResearchMode === undefined) {
    player.autoResearchMode = '...'
}
```

Reply:
I've bypassed the need for an explicit default there: 

https://github.com/Pseudo-Corp/SynergismOfficial/blob/1403ebdcb7e69867939bc3af59273ca3f2fa3e85/src/Toggles.ts#L405-L411

I can add an explicit one in `CheckVariables.ts` if you'd like

Reply:
Fixed by 7050fb2